src/main/obsRecorder.ts

```diff
@@ -352,17 +361,22 @@ const setupSources = (scene: any, audioInputDeviceId: string, audioOutputDeviceI
     }
   }
 
-  setSetting('Output', 'RecTracks', 0); // Bit mask of used tracks: 1111 to use first four (from available six)
+  currentObsTrack = 1;
```

Comment:
Can we make this code clearer than this? I think currently set to 1 means no audio? I don't have a good understanding of the OBS tracks or how this represents them.

Reply:
1 doesn't mean much by itself, really. In our context, however, and currently it does mean that because we add the input source as the first track (1) and audio sources as tracks 2, 3, 4, 5, etc.

'1' is just the "beginning" as it's a 1-based index.